src/SUMMARY.md

```diff
@@ -1,3 +1,9 @@
 # Summary
 
 [CHERI Frequently Asked Questions](cover/README.md)
+- [General](QUESTIONS.md#General)
+- [Operating systems](QUESTIONS.md#Operating-systems)
+    - [CheriBSD](QUESTIONS.md#CheriBSD)
+- [Software porting](QUESTIONS.md#Software-porting)
+- [Compartmentalization](QUESTIONS.md#Compartmentalization)
+- [Community](QUESTIONS.md#Community)
```

Comment:
mdBook doesn't allow to reference sections in the summary. Using `#foo` results in creating a file with this name and `mdbook` constantly prints:
> 2023-06-01 14:50:44 [INFO] (mdbook::cmd::serve): Files changed: ["/Users/kwitaszczyk/Work/repos/elisehaldane/cheri-faq/docs/summary/src/QUESTIONS.md#General", "/Users/kwitaszczyk/Work/repos/elisehaldane/cheri-faq/docs/summary/src/QUESTIONS.md#Operating-systems", "/Users/kwitaszczyk/Work/repos/elisehaldane/cheri-faq/docs/summary/src/QUESTIONS.md#CheriBSD", "/Users/kwitaszczyk/Work/repos/elisehaldane/cheri-faq/docs/summary/src/QUESTIONS.md#Software-porting", "/Users/kwitaszczyk/Work/repos/elisehaldane/cheri-faq/docs/summary/src/QUESTIONS.md#Compartmentalization", "/Users/kwitaszczyk/Work/repos/elisehaldane/cheri-faq/docs/summary/src/QUESTIONS.md#Community"]

I'd suggest https://github.com/CTSRD-CHERI/cheri-faq/pull/5 instead.

Reply:
I'd agree. I'll close this PR in favour of splitting `src/QUESTIONS.md`, as described in [#5](https://github.com/CTSRD-CHERI/cheri-faq/pull/5).